.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\src",
+    "\\src\\views",
+    "\\src\\views\\ApartmentApp",
+    "\\src\\views\\ApartmentApp\\components",
+    "\\src\\views\\ApartmentApp\\components\\StudentApplication",
+    "\\src\\views\\ApartmentApp\\components\\StudentApplication\\components",
+    "\\src\\views\\ApartmentApp\\components\\StudentApplication\\components\\HallSelection",
+    "\\src\\views\\ApartmentApp\\components\\StudentApplication\\components\\HallSelection\\components",
+    "\\src\\views\\ApartmentApp\\components\\StudentApplication\\components\\HallSelection\\components\\HallListItem"
+  ],
+  "SelectedNode": "\\src\\views\\ApartmentApp\\components\\StudentApplication\\index.js",
+  "PreviewInSolutionExplorer": true
+}
```

Comment:
I looks like these files in .vs should be excluded from the repo, since they are local Visual Studio config files.
You can use something like `git update-index --skip-worktree ./.vs/` to make them locally ignored if you plan to continue using Visual Studio instead of VSCode, which might still require you to manually remove them via something like `git rm -r --cached ./.vs/`, which will remove them from the commit but should not actually delete the files from your computer